hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/AbstractHazelcastCacheRegionFactory.java

```diff
@@ -35,20 +34,22 @@
 import org.hibernate.cache.spi.support.StorageAccess;
 import org.hibernate.engine.spi.SessionFactoryImplementor;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Map;
 import java.util.Properties;
 
-import static com.hazelcast.hibernate.CacheEnvironment.getCacheCleanup;
 import static java.lang.Class.forName;
 
 /**
  * Simple RegionFactory implementation to return Hazelcast based local Region implementations
  */
 public abstract class AbstractHazelcastCacheRegionFactory extends RegionFactoryTemplate {
 
-    protected CleanupService cleanupService;
     protected HazelcastInstance instance;
 
+    protected final List<LocalRegionCache> caches = new ArrayList<>();
```

Comment:
nit: localCaches or localRegions or something would be nice as HazelcastCacheRegionFactory also extends this.